pegasus/sites.v3/code.org/views/learn_carousel_block.haml

```diff
@@ -27,7 +27,7 @@
           %br
           %a{href: short_url, style: "font-family: 'Gotham 4r', sans-serif;", target: "_blank"}
             = short_url
-        #gobutton{style: "position: relative;"}
+        %div{id: "gobutton-#{id}", style: "position: relative;"}
           %a{href: "/api/hour/begin/#{launch_code}", target: "_blank"}
```

Comment:
what do these point to if the javascript fails?  (some people have javascript execution disabled; rare, but there are probably other ways JS can fail to execute too.)